aws/table_aws_macie2_classification_job.go

```diff
@@ -161,6 +162,15 @@ func listMacie2ClassificationJobs(ctx context.Context, d *plugin.QueryData, _ *p
 			return !isLast
 		},
 	)
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok {
+			// Throws AccessDeniedException: Macie is not enabled. when AWS Macie is not enabled in a region
+			if awsErr.Message() == "Macie is not enabled." {
```

Comment:
Should we mention why we are validating the `awsErr.Message()`? This will help developers to understand or at least to know that, AWS API response has the same exception code for 2 different cases.